grakn-graql/src/main/java/ai/grakn/graql/internal/hal/HALConceptData.java

```diff
@@ -276,7 +282,7 @@ private void generateTypeEmbedded(Representation halResource, Type type, int sep
                 halResource.withRepresentation(ISA_EDGE, instanceResource);
             });
         }
-        type.subTypes().stream().forEach(instance -> {
+        type.subTypes().forEach(instance -> {
```

Comment:
IntelliJ sees them, so we just need to actually run that inspect code thing again :P